docs/contribute.md

```diff
@@ -203,6 +202,7 @@ flytectl demo start
 
 ### Testing the `basics` project examples on a local demo cluster
```

Comment:
we will need a different example here, since the basics docs have been moved to flyte